cineast-api/src/main/java/org/vitrivr/cineast/api/websocket/handlers/BooleanLookupMessageHandler.java

```diff
@@ -0,0 +1,35 @@
+package org.vitrivr.cineast.api.websocket.handlers;
+
+import org.eclipse.jetty.websocket.api.Session;
+import org.vitrivr.cineast.api.messages.lookup.BooleanLookup;
+import org.vitrivr.cineast.api.messages.result.BooleanLookupResult;
+import org.vitrivr.cineast.api.websocket.handlers.abstracts.StatelessWebsocketMessageHandler;
+import org.vitrivr.cineast.core.db.dao.reader.BooleanReader;
+import org.vitrivr.cineast.standalone.config.Config;
+
+import java.util.List;
+
+public class BooleanLookupMessageHandler  extends StatelessWebsocketMessageHandler<BooleanLookup> {
```

Comment:
At least judging from this double-space, you haven't run the code yet through a formatter. Please use the provided IntelliJ Code Style as outlined in the README and run Formatting on all your code.